microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension.md

```diff
@@ -1,133 +1,211 @@
+JavaScript debugging is now built in to Visual Studio Code. Starting with Version 1.5.7 of Visual Studio Code, you can debug in Chrome, Microsoft Edge, or Node.js without installing any other extensions. If you debug using Microsoft Edge, you can start Microsoft Edge Developer Tools with the JavaScript debugger.
+1. To start a session:
+    * Choose **F5**, or on the menu bar choose the **Debug** icon and choose **Run and debug**.
+    * Open the Visual Studio Code command palette and choose **Debug: Open Link**.
+
+    :::image type="complex" source="./media/edge-devtools-for-vscode-start-session.png" alt-text="Start Microsoft Edge Developer Tools with the JavaScript debugger" lightbox="./media/edge-devtools-for-vscode-start-session.png":::
+       Start Microsoft Edge Developer Tools with the JavaScript debugger  
+    :::image-end:::
+
+1. Choose **Edge**.
+    On the debug toolbar, notice the new Inspect button 
+
+    :::image type="complex" source="./media/edge-devtools-for-vscode-inspect-button.png" alt-text="Inspect button now displayed on debug toolbar" lightbox="./media/edge-devtools-for-vscode-inspect-button.png":::
+       Inspect button now displayed on debug toolbar 
+    :::image-end:::
+
+1. Choose **Inspect** to open Microsoft Edge Developer Tools inside Visual Studio Code.
+    The first time you choose **Inspect**, the editor prompts you to install the [Microsoft Edge Developer Tools for Visual Studio Code extension][https://marketplace.visualstudio.com/items?itemName=ms-edgedevtools.vscode-edge-devtools].
+
+    After installation is complete, when you choose **Inspect** it opens Microsoft Edge Developer Tools inside Visual Studio Code.
+
+    :::image type="complex" source="./media/edge-devtools-for-vscode-tools-inside.png" alt-text="Inspect button opens Microsoft Edge Developer Tools inside Visual Studio Code" lightbox="./media/edge-devtools-for-vscode-tools-inside.png":::
+       Inspect button opens Microsoft Edge Developer Tools inside Visual Studio Code
+    :::image-end:::
+
+    Now you can inspect the DOM, change CSS, and see network requests of your project running in the browser without leaving Visual Studio Code.
     
-    ```json
+    You can also use the Debug Console in the editor to interact with the document in the browser. You have full access to the window object and can use the [Console Utilities API][https://docs.microsoft.com/microsoft-edge/devtools-guide-chromium/console/utilities].
+
+    :::image type="complex" source="./media/edge-devtools-for-vscode-debug-console.png" alt-text="Debug Console in editor interacts with document open in browser" lightbox="./media/edge-devtools-for-vscode-debug-console.png":::
+       Debug Console in editor interacts with document open in browser  
+    :::image-end:::
+
+1. To automatically attach to Microsoft Edge and launch Developer Tools in Visual Studio Code, create a [launch.json][https://code.visualstudio.com/Docs/editor/debugging#_launch-configurations] file.
```

Comment:
can we fix the link?

Reply:
can we change to...

create a launch.json file. For more information, navigate to...